searx/engines/mwmbl.py

```diff
@@ -17,15 +17,14 @@
 
 about = {
     "website": 'https://github.com/mwmbl/mwmbl',
-    "official_api_documentation": 'https://api.mwmbl.org/docs',
     "use_official_api": True,
     "require_api_key": False,
     "results": 'JSON',
 }
 paging = False
 categories = ['general']
 
-api_url = "https://api.mwmbl.org"
+api_url = "https://api.mwmbl.org/api/v1/search/"
```

Comment:
Please remove the trailing `/`. In line 31 there would now be two of them right next to wach other. Also, `search` should probably be removed here because it's already in line 31. Have you even twsted this?